### DISCUSSION TITLE
Distribute `diesel` CLI on deb/rpm

### DISCUSSION THREAD

**Discussion by tgross35** [diesel-rs/diesel#3572] [Ideas]
It seems like diesel ships with homebrew already - are there any blockers (aside from work) to also being available on the debian/redhat/alpine repos?

**Comment by weiznich** [comment]
There are no blockers for distributing diesel cli via deb/rpm, we just do not have the capacity for such things. The homebrew distribution is not maintained by the diesel team, it's done by someone else. If someone is willing to maintain these packages feel free to do that.
